src/main/java/org/jitsi/impl/protocol/xmpp/ChatRoomImpl.java

```diff
@@ -889,21 +889,25 @@ private ChatMemberImpl addMember(String participant)
     {
         ChatMemberImpl newMember;
 
-        if (members.containsKey(participant))
+        synchronized (members)
```

Comment:
I'm not sure I agree that we really need that. addMember is called from one place it should remain like that forever.